project_proposal/tex/mainpage.tex

```diff
@@ -176,7 +176,9 @@ \subsection{Supplementary Requirements}
 \end{table} 
 
 \section{Stakeholders}
-For this project the group does not have any external stakeholders, which is a
+For this project the group does not have any external stakeholders because 
+they don't have any opportunity to communicate with Reflevsbæk and therefore 
+Kristian Nymann Jakobsen has been elected as Project owner, which is a
```

Comment:
I think you need to read this sentence again, and either make a full stop around Kristian, or drop the "which is a problem..." part.

Reply:
can we say "Kristian. Which..."